{
	"name": "CVE-2019-9152",
	"threat_severity": "Moderate",
	"public_date": "2019-02-19T00:00:00Z",
	"bugzilla": {
		"description": "hdf5: out of bounds read in function H5MM_xstrdup in H5MM.c",
		"id": "1684488",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1684488"
	},
	"cvss3": {
		"cvss3_base_score": "5.3",
		"cvss3_scoring_vector": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:L/I:L/A:L",
		"status": "draft"
	},
	"cwe": "CWE-125",
	"details": [
		"An issue was discovered in the HDF HDF5 1.10.4 library. There is an out of bounds read in the function H5MM_xstrdup in H5MM.c when called from H5O_dtype_decode_helper in H5Odtype.c."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2019-9152\nhttps://nvd.nist.gov/vuln/detail/CVE-2019-9152"
	],
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:openstack:10",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 10 (Newton)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:13",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 13 (Queens)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:14",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 14 (Rocky)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:15",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 15 (Stein)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:8",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 8 (Liberty)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:9",
			"fix_state": "Will not fix",
			"package_name": "hdf5",
			"product_name": "Red Hat OpenStack Platform 9 (Mitaka)"
		}
	],
	"csaw": false
}
